
[![Gentoo](https://assets.gentoo.org/tyrian/v2/site-logo.png)](/ "Back to the homepage")
Security

[**Get Gentoo!**](https://get.gentoo.org/)
gentoo.org sites
[gentoo.org](https://www.gentoo.org/ "Main Gentoo website")
[Wiki](https://wiki.gentoo.org/ "Find and contribute documentation")
[Bugs](https://bugs.gentoo.org/ "Report issues and find common issues")
[Forums](https://forums.gentoo.org/ "Discuss with the community")
[Packages](https://packages.gentoo.org/ "Find software for your Gentoo")

[Planet](https://planet.gentoo.org/ "Find out what's going on in the developer community")
[Archives](https://archives.gentoo.org/ "Read up on past discussions")
[Sources](https://sources.gentoo.org/ "Browse our source code")

[Infra Status](https://infra-status.gentoo.org/ "Get updates on the services provided by Gentoo")

* [Home](/)
* [Stay informed](/subscribe)
* [Advisories](/glsa)

# jupyter\_core: Arbitrary Code Execution — GLSA **202301-04**

A vulnerability has been discovered in jupyter\_core which could allow for the execution of code as another user.

### Affected packages

| Package | **dev-python/jupyter\_core** on all architectures |
| --- | --- |
| Affected versions | < **4.11.2** |
| Unaffected versions | >= **4.11.2** |

### Background

jupyter\_core contains core Jupyter functionality.

### Description

jupyter\_core trusts files for execution in the current working directory without validating ownership of those files.

### Impact

By writing to a directory that is used a the current working directory for jupyter\_core by another user, users can elevate privileges to those of another user.

### Workaround

There is no known workaround at this time.

### Resolution

All jupyter\_core users should upgrade to the latest version:

```
 # emerge --sync
 # emerge --ask --oneshot --verbose ">=dev-python/jupyter_core-4.11.2"

```
### References

* [CVE-2022-39286](https://nvd.nist.gov/vuln/detail/CVE-2022-39286)

**Release date**

January 11, 2023

**Latest revision**

January 11, 2023: 1

**Severity**

high

**Exploitable**

remote

**Bugzilla entries**

* [878497](https://bugs.gentoo.org/show_bug.cgi?id=878497)

### Questions or comments?

Please feel free to contact us.

**© 2001–2020 Gentoo Foundation, Inc.**

